mmedit/datasets/pipelines/crop.py

```diff
@@ -84,6 +100,131 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class RandomResizedCrop(object):
+    """Crop data to random size and aspect ratio. Modified keys are
+       the attributes specified in "keys".
```

Comment:
`Modified keys are the attributes specified in "keys".` should be put at the next paragraph. See [here](https://github.com/open-mmlab/mmediting/blob/12905a31694b4794f460694ee3e10600f616265e/mmedit/datasets/pipelines/crop.py#L582).